src/main.js

```diff
@@ -11,6 +11,59 @@ const LIST_PARTICIPANTS_COMMAND = "/teilnehmer";
 const SIGN_OUT_COMMAND = "/austragen";
 const ADD_GUEST = "/gast";
 
+function respondEphemeral(textOutput, responseUrl) {
+  let requestOptions = {
+    method: "post",
+    headers: {
+      "Content-type": "application/json",
+    },
+    payload: JSON.stringify({
+      text: textOutput,
+      response_type: "ephemeral"
+    })
+  };
+  UrlFetchApp.fetch(responseUrl, requestOptions);
```

Comment:
`responseUrl` is a web hook valid for ~30minutes, provided by the initial slack request data parameters.